nixos/modules/installer/tools/nixos-install.sh

```diff
@@ -71,6 +71,17 @@ if ! test -e "$mountPoint"; then
     exit 1
 fi
 
+# Verify permissions are okay-enough
+checkPath="$mountPoint"
```

Comment:
i think mountPoint can be relative, so we should run `realpath $mountPoint`.

Reply:
Good call, otherwise that'd indeed just loop forever. Good catch!